migrations/2021-11-23-132840_email_verification/up.sql

```diff
@@ -0,0 +1,16 @@
+-- use defaults from db for local user init
+alter table local_user alter column theme set default 'browser';
+alter table local_user alter column default_listing_type set default 2;
+
+-- add tables and columns for optional email verification
+alter table site add column email_verification boolean not null default false;
+alter table local_user add column activated boolean not null default false;
```

Comment:
Instead of activated, could you rename this `email_verified`, or `verified_email` 

And the site one, `require_email_verification`